ivy/functional/frontends/paddle/stat.py

```diff
@@ -52,6 +52,15 @@ def numel(x, name=None):
     return ivy.array(prod if prod > 0 else ivy.array(length, dtype=ivy.int64))
 
 
+@with_supported_dtypes(
+    {"2.5.1 and below": ("float32", "float64", "int32", "int64")},
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def quantile(x, q, axis=None, keepdim=False):
```

Comment:
you forgot to add the 'name', parameter.

Reply:
I added 'name' parameter